_migration-assistant/migration-phases/planning-your-migration/assessing-your-cluster-for-migration.md

```diff
@@ -14,19 +14,42 @@ The goal of the Migration Assistant is to streamline the process of migrating fr
 
 ## Understanding breaking changes
 
-Before performing any upgrade or migration, you should review any documentation of breaking changes.  Even if the cluster is migrated there might be changes required for clients to connect to the new cluster
-
-## Upgrade and breaking changes guides
-
-For migrations paths between Elasticsearch 6.8 and OpenSearch 2.x users should be familiar with documentation in the links below that apply to their specific case:
-
-* [Upgrading Amazon Service Domains](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/version-migration.html).
-
-* [Changes from Elasticsearch to OpenSearch fork](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/rename.html).
-
-* [OpenSearch Breaking Changes]({{site.url}}{{site.baseurl}}/breaking-changes/).
-
-The next step is to set up a proper test bed to verify that your applications will work as expected on the target version.
+Before performing any upgrade or migration, you should review any breaking changes that might exist between version, because there might be changes required for clients to connect to the new cluster:
+
+<link rel="stylesheet" href="{{site.url}}{{site.baseurl}}/migration-assistant/assets/css/breaking-changes-selector.css">
+
+<div class="breaking-changes-selector">
+  <h4>Find breaking changes for your migration path</h4>
```

Comment:
```suggestion
  <h4>See a list of breaking changes for your migration path</h4>
```

Reply:
@natebower: This test introduces a dropdown where you can find a list of breaking changes between versions of ElasticSearch and OpenSearch. With that in mind, does my suggestion make sense to you?